packages/grafana-toolkit/src/config/jest.plugin.config.ts

```diff
@@ -80,3 +94,16 @@ export const jestConfig = (baseDir: string = process.cwd()) => {
     ...otherOverrides,
   };
 };
+
+/**
+ * This will load the existing just setup, or use the default if it exists
+ */
+export const loadJestPluginConfig = (baseDir: string = process.cwd()) => {
+  const cfgpath = path.resolve(baseDir, 'jest.config.js');
+  if (!fs.existsSync(cfgpath)) {
```

Comment:
It's a best practice to always use async functions when possible. Any defined function can then be used concurrently.

Reply:
I am nervous about requiring the whole call stack to become async... all to support somehting that is identical and should not be called async

Reply:
Okay, can be fixed later